local_server/local_tts_server/start_kokoro_server.ps1

```
@@ -0,0 +1,516 @@
+        foreach ($connection in $connections) {
+            if ($connection.LocalAddress -eq $HostAddress -or $connection.LocalAddress -in @("0.0.0.0", "::")) {
+                $process = Get-Process -Id $connection.OwningProcess -ErrorAction SilentlyContinue
+                return [pscustomobject]@{
+                    Pid = $connection.OwningProcess
+                    Name = if ($process) { $process.ProcessName } else { "<unknown>" }
+                    Path = if ($process) { $process.Path } else { "" }
+                }
+            }
+        }
+    } catch {
+    }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)